src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/boss_bronjahm.cpp

```diff
@@ -176,6 +201,8 @@ class boss_bronjahm : public CreatureScript
 
                 DoMeleeAttackIfReady();
             }
+        private:
+            uint32 _soulFragmentsSpawned;
```

Comment:
> **jackpoz** added a note on [8 Dec 2014](https://github.com/TrinityCore/TrinityCore/pull/13606/files#r21454627)
please initialize all fields in the constructor

Sorry, I don't know about constructors. Is that part of the definitions in the header?